float32 mission_num
boolean is_waiting_time